### PR TITLE
Add package json with kalamuna namespacing

### PR DESCRIPTION
**Pull request by thiagodemellobueno**
This way we can load this package in as an npm package in other projects

### CODE REVIEW

**Review by RobLoach — APPROVED** [review]
Doesn't really need the dependencies, but they work just fine :+1:

**Comment by thiagodemellobueno** [comment]
@RobLoach  - I added some basics so this repo could build it's own styleguide.
That way developing here isn't guesswork.

Also some of the modules (specifically unordered list) were calling on extendables that didnt exist.

That complexifies this repo a bit, but doesn make it more 'just works'

**Comment by thiagodemellobueno** [comment]
This also adds a video atom.

File: src/components/atoms/video/_video.scss [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+Video component utilizing video.js
+
+Markup: video.html.twig
+
+Styleguide media.video
+
+*/
+
+@import "http://vjs.zencdn.net/6.2.0/video.js"
```
Comment (soniktrooth) [inline src/components/atoms/video/_video.scss]: @thiagodemellobueno Importing a js file into a css file? Seems legit.

Comment (RobLoach) [reply src/components/atoms/video/_video.scss]: Works with Webpack!

**Review by soniktrooth — APPROVED** [review]
woot

**Pull request merged**